addons/scratchr2/forums.css

```diff
@@ -337,10 +337,10 @@ body > #pagewrapper {
 .my-ocular-reaction-button,
 .my-ocular-reaction-button:hover {
   color: var(--darkWww-gray-text, #575e75);
-  box-shadow: 0 0 2px #0fbd8c;
+  box-shadow: 0 0 2px #328554;
 }
 .my-ocular-reaction-button.selected {
-  background-color: #0fbd8c;
+  background-color: #328554;
   color: white;
 }
```

Comment:
ocular's brand color is still `#0fbd8c`. Text readability is important though, and the goal of the 2.0 → 3.0 addon is to be consistent with Scratch, so I think this is a good change.